client/src/components/Footer.js

```diff
@@ -4,60 +4,44 @@ import Dice from '../icons/Dice';
 import AddQuestion from '../icons/AddQuestion';
 import ButtonLink from './ButtonLink';
 import Highscore from '../icons/HighScore';
+import {
+  linearGradientBoxShadow,
+  flexRowWrapCenter,
+  noBorderOutlineBGTextDeco
+} from '../styles/General';
 
 const FooterBar = styled.div`
-  position: absolute;
-  bottom: 0px;
+  ${linearGradientBoxShadow};
   display: flex;
   justify-content: space-around;
   align-items: center;
-  position: absolute;
   bottom: 0px;
+  position: absolute;
```

Comment:
The `FooterBar` doesn't have to have `position: absolute`, if the parent element of main layout elements (such as `main`, `header` and `footer`) is a flexbox you can give `footer` and `header` a fixed height and the `main` element a `flex-grow: 1`

see `main`, `header` and `footer` on this [exercise](https://github.com/marwinburesch/time.wise/blob/solution-html/style.scss ) for an example.

Reply:
true, but is there any downside of it?

Reply:
Not really, unless it doesn't fit your app